packages/neo-one-smart-contract-lib/src/__data__/contracts/crowdsale/TestCrowdsaleContract.ts

```diff
@@ -0,0 +1,18 @@
+import { Address, Deploy, Fixed } from '@neo-one/smart-contract';
```

Comment:
Can we unnest these? I think just setting them in the `__data__/contracts` folder is fine.

Reply:
I see now you had it unnested but deleted that change? Curious why :thinking:

Reply:
keeping it in alignment with the folder structure of the __tests__ and /src/
based on this:
https://github.com/OpenZeppelin/openzeppelin-solidity/tree/master/contracts/crowdsale